metadata.yaml

```
@@ -33,3 +35,4 @@ versions:
   - sha: dffc69779a22e436dddbf9f11524ff297a8f1949
     changeNotes: Includes version 0.1 of Octanist, featuring essential functionalities for lead qualification and offline conversion tracking. Captures data such as GCLID, company name, email, phone number, website/domain, and page path.
   
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (1 > 0)

(empty-lines)